src/fu-tool.c

```diff
@@ -926,6 +932,12 @@ fu_util_update_device_changed_cb(FwupdClient *client, FwupdDevice *device, FuUti
 static void
 fu_util_display_current_message(FuUtilPrivate *priv)
 {
+	/* print all POST requests */
+	for (guint i = 0; i < priv->post_requests->len; i++) {
+		FwupdRequest *request = g_ptr_array_index(priv->post_requests, i);
+		g_print("%s\n", fwupd_request_get_message(request));
+	}
+
 	if (priv->current_message == NULL)
```

Comment:
I am not sure what to do with the `current_message`; should we drop it?

Reply:
Yup, I think so; I think the behaviour between fu-util.c and fu-tool.c should be exactly the same.

Reply:
Also, what is the purpose of `fu-util-common.c`? if the function is copied from `src/fu-util.c` to `src/fu-tool.c` and it is strickly identical, it should be moved and shared to `fu-util-common.c`, right?

Reply:
> it should be moved and shared to fu-util-common.c, right

Ya, but only if it's general purpose. Don't go overboard.